### PR TITLE
Gradebook endpoints v2

### LINKED ISSUE #86: Retrieve session grading info endpoint

### PR DESCRIPTION
**Pull request by vutoan1245**
Resolve #86

Improve query for gradebook.
Return example:
```json
{
    "sessions": [
        {
            "id": 1,
            "name": "Session 1"
        },
        {
            "id": 2,
            "name": "Session 2"
        }
    ],
    "students": [
        {
            "name": "Student1",
            "canvasId": 2,
            "SessionGrades": [
                {
                    "id": 1,
                    "points": 8,
                    "maxPoints": 10
                },
                {
                    "id": 3,
                    "points": 9.3,
                    "maxPoints": 10
                }
            ]
        },
        {
            "name": "Student2",
            "canvasId": 6,
            "SessionGrades": [
                {
                    "id": 2,
                    "points": 8,
                    "maxPoints": 10
                },
                {
                    "id": 4,
                    "points": 7,
                    "maxPoints": 10
                }
            ]
        }
    ]
}
```

### CODE REVIEW

**Review by Goff-Davis — APPROVED** [review]
LGTM but LJ should prob take a look before it's merged

**Review by ljoks — APPROVED** [review]
can't really test it cus i don't have a canvas instance to get students from but the code LGTM

**Pull request merged**
